### DISCUSSION TITLE
i need help my facebook has been hacked i am disabled

### DISCUSSION THREAD

**Discussion by Twb40** [community/community#127917] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

please help

**Comment by CodeModeYT** [comment]
### I have two questions.
1. Why did you ask this on here?
2. How do you expect to get an answer with providing no additional information? (Not even regarding the fact that you are on the wrong platform for this question)

### I am still gonna try to help you though:
If you can still get into your account, immediately change your password and if you also used the same email/password combination on other platforms, immediately change it there too.
After that you should contact Facebook support with a *more precise* (!!!) description of your problem.

Hope this helps.

**Reply by Twb40** [reply]
Hi sorry if I went on the wrong place .I started to follow  what was on my pc that is why I ended up  here.  I went on here to find out who was flying drones over my house.Thanks for your help. Plus it showed who was testing out the security issues   meta has.

**Comment by Twb40** [comment]
Thank you

Sent from AOL on Android 
 
  On Sun, Jun 9, 2024 at 2:24 PM, ***@***.***> wrote:   


I have two questions.
   
   - Why did you ask this on here?
   - How do you expect to get an answer with providing no additional information? (Not even regarding the fact that you are on the wrong platform for this question)

I am still gonna try to help you though:

If you can still get into your account, immediately change your password and if you also used the same email/password combination on other platforms, immediately change it there too.
After that you should contact Facebook support with a more precise (!!!) description of your problem.

Hope this helps.

**Reply by Twb40** [reply]
Thank u for your time.i was flowing what was on my pc.I thought I could get help from the best people  who know computers.
